cli/templatecreate.go

```diff
@@ -46,39 +47,22 @@ func (r *RootCmd) templateCreate() *clibase.Cmd {
 			r.InitClient(client),
 		),
 		Handler: func(inv *clibase.Invocation) error {
-			isTemplateSchedulingOptionsSet := failureTTL != 0 || dormancyThreshold != 0 || dormancyAutoDeletion != 0 || maxTTL != 0
-
-			if isTemplateSchedulingOptionsSet || requireActiveVersion {
-				if failureTTL != 0 || dormancyThreshold != 0 || dormancyAutoDeletion != 0 {
-					// This call can be removed when workspace_actions is no longer experimental
-					experiments, exErr := client.Experiments(inv.Context())
-					if exErr != nil {
-						return xerrors.Errorf("get experiments: %w", exErr)
-					}
-
-					if !experiments.Enabled(codersdk.ExperimentWorkspaceActions) {
-						return xerrors.Errorf("--failure-ttl, --dormancy-threshold, and --dormancy-auto-deletion are experimental features. Use the workspace_actions CODER_EXPERIMENTS flag to set these configuration values.")
-					}
-				}
-
-				entitlements, err := client.Entitlements(inv.Context())
-				if cerr, ok := codersdk.AsError(err); ok && cerr.StatusCode() == http.StatusNotFound {
-					return xerrors.Errorf("your deployment appears to be an AGPL deployment, so you cannot set enterprise-only flags")
-				} else if err != nil {
-					return xerrors.Errorf("get entitlements: %w", err)
-				}
-
-				if isTemplateSchedulingOptionsSet {
-					if !entitlements.Features[codersdk.FeatureAdvancedTemplateScheduling].Enabled {
-						return xerrors.Errorf("your license is not entitled to use advanced template scheduling, so you cannot set --failure-ttl, --inactivity-ttl, or --max-ttl")
-					}
-				}
-
-				if requireActiveVersion {
-					if !entitlements.Features[codersdk.FeatureAccessControl].Enabled {
-						return xerrors.Errorf("your license is not entitled to use enterprise access control, so you cannot set --require-active-version")
-					}
-				}
+			_, _ = fmt.Fprintln(inv.Stdout, "\n"+pretty.Sprint(cliui.DefaultStyles.Wrap,
+				pretty.Sprint(
+					cliui.DefaultStyles.Warn, "DEPRECATION WARNING: The `coder templates push` command should be used instead. This command will be removed in a future release. ")+"\n"))
+			time.Sleep(1 * time.Second)
```

Comment:
Love the sleep-based incentive, but we should tell the user that it's sleeping for a moment in the deprecation message so that the user doesn't just think our CLI is slow.